travel/src/main/java/com/tx/travel/controller/CostCenterController.java

```diff
@@ -0,0 +1,112 @@
+package com.tx.travel.controller;
+
+import com.tx.travel.mapper.CostCenterMapperImplementation;
+import com.tx.travel.model.CostCenter;
+import com.tx.travel.payload.request.CostCenterRequest;
+import com.tx.travel.payload.response.CostCenterResponse;
+import com.tx.travel.service.CostCenterService;
+import com.tx.travel.service.CostCenterServiceImplementation;
+import com.tx.travel.service.exception.CostCenterCodeAlreadyExists;
+import com.tx.travel.service.exception.CostCenterNotPresent;
+import java.util.ArrayList;
+import java.util.List;
+import javax.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+
+@RestController
+@RequestMapping("/api/cost-centers")
```

Comment:
have a look at this [comment](https://github.com/tx-services-Internship/travel-java-backend-monolith-spyro/pull/4#discussion_r1031729324)